Web/Edubase.Web.UI/Areas/Governors/Models/GovernorsGridViewModel.cs

```diff
@@ -106,11 +107,24 @@ public GovernorsGridViewModel()
         private void CreateGrids(GovernorsDetailsDto dto, IEnumerable<GovernorModel> governors, bool isHistoric,
             int? groupUid, int? establishmentUrn)
         {
-            var roles = dto.ApplicableRoles.Where(role => !EnumSets.eSharedGovernorRoles.Contains(role)
-                                                          ||
-                                                          (RoleEquivalence.GetLocalEquivalentToSharedRole(role) != null
-                                                           && !dto.ApplicableRoles.Contains(RoleEquivalence
-                                                               .GetLocalEquivalentToSharedRole(role).Value)));
+            var roles = dto.ApplicableRoles.Where(role =>
+            {
+                if (!EnumSets.eSharedGovernorRoles.Contains(role))
+                {
+                    return true;
+                }
+
+                var localEquivalent = RoleEquivalence.GetLocalEquivalentToSharedRole(role);
+                if (localEquivalent != null)
+                {
+                        if (!dto.ApplicableRoles.Contains(localEquivalent.Value))
+                        {
+                            return true;
+                        }
```

Comment:
Missing the `else { return false; }` in this refactor?

Reply:
the else { return false; } restricts the roles so the additional governor is not seen - the way it is now is the opposite and allows more than we need, so this has been refactored